zcash_client_backend/Cargo.toml

```diff
@@ -154,6 +154,8 @@ zcash_proofs = { workspace = true, features = ["bundled-prover"] }
 zcash_protocol = { workspace = true, features = ["local-consensus"] }
 
 [features]
+default = ["time/default"]
+
```

Comment:
Is `time/default` actually used anywhere by this crate? If not, we can remove this in the breaking change release immediately after this PR merges (which could potentially be the release this goes into).